src/pages/Diet/components/AddImage.jsx

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+export function AddImage() {
+  let [link, setLink] = useState('');
+  let [files, setFiles] = useState(null);
+
+  function handleFileChange(e) {
+    const { files } = e.target;
+    setFiles(files[0]);
+
+    if (files.length === 0) {
+      setLink('');
+    } else {
+      const imageURL = window.URL.createObjectURL(files[0]);
+      setLink(imageURL);
+    }
+  }
+
+  async function submitImg() {
+    try {
+      console.log('files', files, 'link', link);
+      let form = new FormData();
+      form.append('image', files);
+      form.append('album', 'PZijGmD');
+
+      const res = await axios({
+        method: 'POST',
+        url: 'https://api.imgur.com/3/image',
+        headers: {
+          Authorization: `Bearer ${`Access Token`}`,
```

Comment:
remember to add `REACT_APP_IMGUR_ACCESS_TOKEN` in `.env` file

Reply:
Is that safe to add token in `.env` file?
I thought the file wouldn't be shown on gitHub, but it would.